_includes/social.html

```diff
@@ -0,0 +1,13 @@
+<ul class="social-media-list">
```

Comment:
I really wish we could have a hash of site => domain and then could iterate through e.g.,

```liquid
%{ for site_hash in site.social_sites %}
  {% if site.social[site_hash[0]] %}
    <li>
      <a href="https://{{ site_hash[1] }}/{{ site.social[site_hash[0]] | cgi_escape | escape }}">
        <svg class="svg-icon"><use xlink:href="/assets/icons.svg#{{ site_hash[0] }}"></use></svg> 
        <span class="username">{{  site.social[site_hash[0]] | escape }}</span>
      </a>
    </li>
  {% endif %} 
{% endfor %}
```